resources/job.rb

```diff
@@ -0,0 +1,9 @@
+actions :create, :delete
+default_action :create
+
+attribute :name, name_attribute: true, regex: /^[-_+.a-zA-Z0-9]+$/
+attribute :project, kind_of: String, required: true
+attribute :config, kind_of: Hash, default: {}
+
+attribute :endpoint, kind_of: String, default: 'https://localhost'
```

Comment:
provider user can do that if they want. I'd like this provider to keep it simple